api/src/stampy_chat/chat.py

```diff
@@ -238,14 +228,14 @@ def talk_to_robot_internal(index, query: str, mode: str, history: List[Dict[str,
         yield {'state': 'error', 'error': str(e)}
 
 # convert talk_to_robot_internal from dict generator into json generator
-def talk_to_robot(index, query: str, mode: str, history: List[Dict[str, str]], k: int = STANDARD_K, log: Callable = logger.info):
```

Comment:
Ah I'm seeing you caught this here. Might be better to do these changes on #86 and then merge from `log-queries` up into `mysql-logs`.